package/src/index.ts

```diff
@@ -70,6 +92,260 @@ export class CandleClient {
   }): Promise<string> {
     return this.candle.executeTool(tool);
   }
+
+  public async getAssetAccounts(
+    query: AssetAccountQuery = {
+      linkedAccountIDs: undefined,
+      assetKind: undefined,
+    }
+  ): Promise<AssetAccount[]> {
+    const accounts = await this.candle.getAssetAccounts(query);
+    return accounts.map(this.convertToAssetAccount);
```

Comment:
```suggestion
    return accounts.map((_) => this.convertToAssetAccount(_));
```
#req passing around JS class instance functions is dangerous, this would be a linter error with our normal linter settings

Reply:
how is it dangerous?

Reply:
https://typescript-eslint.io/rules/unbound-method